src/app/api/product-reviews/route.ts

```diff
@@ -15,53 +16,60 @@ const requestType = z.object({
 });
 
 export async function POST(request: Request) {
-  let body = undefined;
-  try {
-    body = await request.json();
-  } catch (err) {
-    return response400("Incorect format");
-  }
-  const results = requestType.safeParse(body);
-  if (!results.success!) {
-    return response400("Incorect format");
-  }
-  const parsedBody = results.data;
-
-  try {
-    let product = await getProduct(parsedBody.product_id);
-    if (product === undefined) {
-      return response400("There is no product with this ID");
+  const user = await getUserServerSide();
+  if (user === undefined) {
+    console.log("User not logged in");
```

Comment:
If the user is not logged in that's it. Only a console log? Shouldn't your return a 401?